src/services/maze.service.ts

```diff
@@ -86,6 +89,65 @@ function handleMove(
     return result;
 }
 
+interface MultipleMoveResult {
+    status: Status;
+    map: CellObject[];

```

Comment:
```suggestion
    map: Cell[];
```
sửa lại tên cho anh nha